odoo/tests/common.py

```diff
@@ -1342,7 +1346,8 @@ def delete_cookie(self, name, **kwargs):
         _id = self._websocket_send('Network.deleteCookies', params=params)
         return self._websocket_wait_id(_id)
 
-    def _wait_ready(self, ready_code, timeout=60):
+    def _wait_ready(self, ready_code=None, timeout=60):
+        ready_code = ready_code or "document.readyState === 'complete'"
```

Comment:
This seems simpler IMHO:

```suggestion
    def _wait_ready(self, ready_code="document.readyState === 'complete'", timeout=60):
```

Reply:
Ah, hold on! It wouldn't be backwards compatible. Forget it, sorry.

Reply:
It is not the same think, I want to ensure the behavior remains the same.